src/cloudflare/internal/d1-api.ts

```diff
@@ -227,7 +227,7 @@ class D1PreparedStatement {
   }
 
   public async first<T = unknown>(colName: string): Promise<T | null>
-  public async first<T = unknown>(): Promise<Record<string, T> | null>
+  public async first<T = Record<string, unknown>>(): Promise<T | null>
```

Comment:
OK this looks good. Again this would be a breaking change to any one who has previously used:

```
const result = await statement.first<string>()
```

Since they would have expected a `Record<string, string> | null` but now the type is just `string | null`.
Is that acceptable?

Reply:
Yeah that change only _just_ went out, and was an error. You'd rarely have a table of all the same value types, so I think the only people who've tried to use that type are the ones who pointed out the error!